server/database/repositories/TournamentRepository.ts

```diff
@@ -96,16 +97,20 @@ export default class TournamentRepository {
     }]);
     const tDocs: TournamentDocument[] = await TournamentModel.find().skip(skip).limit(pageSize).exec();
     const tournaments: TournamentWithoutMS[] = tDocs.map((tDoc) => toTournament(tDoc));
-    return { totalRows: totalRows[0].count, tournaments };
+    let totalRowsCount = 0;
+    if (totalRows[0]) totalRowsCount = totalRows[0].count;
+    return { totalRows: totalRowsCount, tournaments };
```

Comment:
```suggestion
    let totalRows = 0;
    if (totalRows[0]) totalRows = totalRowsData[0].count;
    return { totalRows, tournaments };
```
What do you think? It would be easier

Reply:
totalRows is another variable

Reply:
Yep i chacnged this in this suggetion
totalRows => totalRowsData
totalRowsCount => totalRows

It is less important, but I believe it is more readable

Reply:
changed